packages/quantic/force-app/main/default/lwc/quanticSmartSnippetAnswer/quanticSmartSnippetAnswer.js

```diff
@@ -0,0 +1,81 @@
+import {LinkUtils} from 'c/quanticUtils';
+import {LightningElement, api} from 'lwc';
+
+/**
+ * The `QuanticSmartSnippetAnswer` component displays the smart snippet answer.
+ * @category Search
+ * @example
+ *  <c-quantic-smart-snippet-answer answer={answer} actions={actions}></c-quantic-smart-snippet-answer>
+ * @internal
+ */
+export default class QuanticSmartSnippetAnswer extends LightningElement {
+  /**
+   * The smart snippet answer.
+   * @api
+   * @type {string}
+   */
+  @api
+  get answer() {
+    return this._answer;
+  }
+  set answer(value) {
+    this._answer = value;
+    if (!this.isInitialRender) {
+      this.updateSmartSnippetAnswer();
+    }
+  }
+
+  /**
+   * The actions that need to be bound to the links of the smart snippet source.
+   * @api
+   * @type {{select: function, beginDelayedSelect: function, cancelPendingSelect: function  }}
+   */
+  @api actions;
+
+  /** @type {boolean} */
+  isInitialRender = true;
+  /** @type {string} */
+  _answer;
+
+  renderedCallback() {
+    this.updateSmartSnippetAnswer();
+    this.isInitialRender = false;
+  }
+
+  updateSmartSnippetAnswer() {
+    const snippetAnswerElement = this.template.querySelector(
+      '.smart-snippet-answer'
+    );
+    // eslint-disable-next-line @lwc/lwc/no-inner-html
```

Comment:
Do we want eslint to be disabled here?

Reply:
yes, this eslint rule is usually useful cause we don't wanna modify HTML from the Js bundle of our LWC but in this particular case we are using the `lwc:dom="manual"`  directive in the HTML template so we can safely use `innerHTML`.